src/Parsing/ParserState.php

```diff
@@ -131,12 +131,12 @@ public function parseIdentifier($ignoreCase = true)
         if ($result === null) {
             throw new UnexpectedTokenException('', $this->peek(5), 'identifier', $this->lineNumber);
         }
-        $sCharacter = null;
-        while (!$this->isEnd() && ($sCharacter = $this->parseCharacter(true)) !== null) {
-            if (\preg_match('/[a-zA-Z0-9\\x{00A0}-\\x{FFFF}_-]/Sux', $sCharacter)) {
-                $result .= $sCharacter;
+        $character = null;
```

Comment:
The initialization here appears to be unnecessary.  Beyond the scope of this PR.